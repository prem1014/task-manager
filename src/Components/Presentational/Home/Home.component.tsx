import React from 'react';
import { useHistory } from 'react-router-dom';

const Home = () => {
    const history = useHistory();
    const goToCreateTaskPage = (e: any) => {
        e.preventDefault();
        history.replace('/tasks')
    }

    return (
        <React.Fragment>
            <div className="row margin-top-20px">
                <div className="col-12">
                    <h4 className="text-center">Welcome To Task Management Web App</h4>
                    <h5 className="text-center">
                        <a href="#" onClick={(event) => goToCreateTaskPage(event)}>
                            Click Here To Go To Create Task Page
                        </a>
                    </h5>
                </div>
            </div>
        </React.Fragment>
    )
}

export default Home;