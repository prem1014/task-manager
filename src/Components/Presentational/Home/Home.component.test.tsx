import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Home from './Home.component';

describe('Home component test suite', () => {
    Enzyme.configure({ adapter: new Adapter() });
    let componentWrapper: any;

    beforeEach( () => {
        componentWrapper = shallow(<Home />);
    });

    it('home component snapshot test', () => {
        expect(componentWrapper).toMatchSnapshot();
    });
});