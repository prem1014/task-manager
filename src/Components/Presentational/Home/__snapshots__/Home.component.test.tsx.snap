// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test suite home component snapshot test 1`] = `ShallowWrapper {}`;
